nanshe_workflow/data.py

```diff
@@ -88,6 +89,14 @@ def _read_chunk(fn, dn, idx):
     return a
 
 
+def save_tiff(fn, a):
+    if os.path.exists(fn):
+        os.remove(fn)
+    with tifffile.TiffWriter(fn, bigtiff=True) as tif:
+        for i in irange(a.shape[0]):
+            tif.save(numpy.array(a[i]))
```

Comment:
Should use `asarray` to potentially avoid a copy. Fixing in PR ( https://github.com/nanshe-org/nanshe_workflow/pull/186 ).